explorer/src/pages/Attestations/index.tsx

```diff
@@ -36,9 +38,35 @@ export const Attestations: React.FC = () => {
   const where = searchParams.get(EQueryParams.WHERE)
     ? (JSON.parse(searchParams.get(EQueryParams.WHERE) ?? "") as Attestation_filter)
     : undefined;
-
   const [lastID, setLastID] = useState<number>(getItemsByPage(page, itemsPerPage));
 
+  const isByAttestationType = "schema_" in (where || {});
```

Comment:
this condition might be a bottleneck; I would suggest creating two different pages for filtering by schema and for filtering by portals

Reply:
after discussion, we decided to keep one page for all attestations